app/controllers/user_funded_projects_controller.rb

```diff
@@ -5,4 +5,20 @@ def new
     @project = Project.find(params[:project_id])
   end
 
+  def create
+    @user = current_user
+    @project = Project.find(params[:project_id])
+    fund = @user.user_funded_projects.create(project_params)
+    @role = Role.find_or_create_by(name: "project_funder")
+    @user.roles << @role
+    @project.funders << @user
```

Comment:
If you look at the project model there is a relationship that looks like 'has_many: :funders, through: :user_funded_projects, source: :users'